File: netway/main/billing.netway.co.th/public_html/includes/modules/Hosting/rvcpanel_manage2/include/cpl-3.6/perl/examples/Expire_License.pl
#!/usr/bin/perl

use cPanelLicensing;

my $ip = '__IPTOEXPIRE__';

my $licenseManager = new cPanelLicensing(
    user => '__YOU@HOST.COM__',
    pass => '__YOURPASS__'
);

my $liscid = $licenseManager->fetchLicenseId( 'ip' => $ip );
if ( $liscid > 0 ) {
    my $result = $licenseManager->expireLicense(
        'liscid'  => $liscid,
        'reason'  => 'Automatic Removal',
        'expcode' => 'normal'
    );

    print "$result\n";
}
else {
    print "There is no active license for ip address: ${ip}\n";
}

File: rvglobalsoft/main/rvglobalsoft.com/public_html/includes/libs/cpanel/cpl-3.6/php/examples/expire_license.php
<?php
include("../cpl.inc.php");
$cpl = new cPanelLicensing("__USERNAME__", "__PASSWORD__");

$ip = "__IP__";

$lisc = (array)$cpl->fetchLicenseId( array( "ip" => $ip ) );
$liscid = $lisc["@attributes"]["licenseid"];

if ( $liscid > 0 ) {
    $expire = (array)$cpl->expireLicense(array(
        liscid => $liscid,
        reason => "Automagic Expiration",
        expcode => "normal"
        )
    );
    print $expire["@attributes"]["result"] . "\n";
} else {
    print "There is no valid license for $ip\n";
}

?>
